provider/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.5.0'
+__version__ = '0.6.0'
```

Comment:
I've always understood bumping the minor version to be acceptable for breaking changes in pre-1.0 software, but I don't have a strong feeling about the version number.